### PR TITLE
Add openai.embeddings connector

### PR DESCRIPTION
**Pull request by oshadajay**
# Description
Add openai.embeddings connector

Fixes: https://github.com/wso2-enterprise/choreo/issues/20370

### CODE REVIEW

File: openapi/openai.embeddings/Module.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+```ballerina
+public function main() returns error? {
+    embeddings:CreateEmbeddingRequest req = {
+        model: "text-embedding-ada-002",
+        input: "I have bought several of the Vitality canned"
+    };
+    embeddings:CreateEmbeddingResponse res = check embeddingsClient->/embeddings.post(req);
+    io:println(res);
+}
+```
```
Comment (NipunaRanasinghe) [inline openapi/openai.embeddings/Module.md:49]: Let's indent this block to be aligned with the numbered list.

Comment (oshadajay) [reply openapi/openai.embeddings/Module.md]: Addressed in https://github.com/ballerina-platform/openapi-connectors/pull/773/commits/1ba11388a777fffdeabaa1a648d299569e7359c4.

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Comment by abeykoon** [comment]
LGTM

**Review by abeykoon — APPROVED** [review]

**Pull request merged**